### PR TITLE
LPS-174994 [PRM] Set State Manager in Activity

### PR DESCRIPTION
**Pull request by Manda-vit**

### CODE REVIEW

**Comment by Manda-vit** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.js [hunk 1]
```diff
@@ -151,6 +151,16 @@ const TypeActivityKey = {
...
 
+const ActivityStatus = {
+	active: 'active',
+	expired: 'expired',
+};
+
+const activityStatusClassName = {
+	[ActivityStatus.active]: 'label label-tonal-success ml-2',
+	[ActivityStatus.expired]: 'label label-tonal-danger ml-2',
+};
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.js:162]: Please try to substitute these enum properties with the Uppercase versions.

**Review by JoMaAlves — CHANGES_REQUESTED** [review]
🛠️ Needs some changes 🛠️

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.js (hunk 1 above)
Comment (Manda-vit) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.js]: OK!

**Review by JoMaAlves — APPROVED** [review]
LGTM ​🦖​🌋​ 
Thank you @Manda-vit

**Comment by JoMaAlves** [comment]
ci:test:relevant

**Comment by JoMaAlves** [comment]
Ninguem da forward nessa PR, ficou faltando o Json do State Manager

**Pull request closed without merging**